doc/reference_measure.xml

```diff
@@ -2598,6 +2598,91 @@ The current implementation supports only vertices as the discrete locations. Thi
 			</programlisting>
 
 	  </refsection>
+          <refsection>
+                <title>See Also</title>
+
+                <para><xref linkend="ST_FrechetDistance" /></para>
+          </refsection>
+	</refentry>
+
+	<refentry id="ST_FrechetDistance">
+	  <refnamediv>
+		<refname>ST_FrechetDistance</refname>
+
+		<refpurpose>Returns the Fréchet distance between two geometries. This is a measure of similarity between curves that takes into account the location
+		and ordering of the points along the curves.  Units are in the units of the spatial reference system of the geometries.</refpurpose>
```

Comment:
Can you add a reference of the algorithm specification, and an explanation of how this is different from Hausdorff ?